odak/learn/tools/matrix.py

```diff
@@ -32,7 +32,7 @@ def zero_pad(field, size=None, method='center'):
     Parameters
     ----------
     field             : ndarray
-                        Input field MxN array.
+                        Input field ...xMxN array.
```

Comment:
These functions require slight modifications for working with color images or batched tensors.

PyTorch convention is for tensors to be in CHW, BCHW, or B...HW format with an arbitrary number of leading axes, but with the spatial dimensions always occupying axes `-2, -1`. I've modified this function to use negative indices to accomodate for arbitrary tensor dimensions